tests/modules/worksheets/test_worksheet.py

```diff
@@ -30,6 +34,38 @@ def worksheet(client, tenant, group):
     return response.data.createWorksheet
 
 
+@pytest.fixture(scope='module', autouse=True)
+def mock_s3_client(module_mocker):
+    s3_client = MagicMock()
+    module_mocker.patch('dataall.modules.worksheets.services.worksheet_query_result_service.S3_client', s3_client)
+
```

Comment:
nit: use autospec=True in mocker.patch method - autospec ensures that only functions defined in the class are patched. For example if in the tests we call the funciton s3_client.do_something it will throw and error with autospec, while with just MagicMock it won't

Reply:
changes are done